apps/deno/tests/test14.ts

```diff
@@ -0,0 +1,71 @@
+import { Assets } from '../index.ts';
+import { PageSizes, PDFDocument, BlendMode, degrees, rgb } from '../../../dist/pdf-lib.esm.js';
+
+
+const modeNames = Object.values<string>(BlendMode);
+const objectSize = 80;
+const numObjects = 60;
+
+export default async (_assets: Assets) => {
```

Comment:
This is a nice test (I enjoy the output). But please see my previous comment about testing the `blendMode` option in the drawing operations of existing tests. Also, it's important to test the `blendMode` option on every draw method that supports it. We're currently only testing the `drawCircle`, `drawSquare`, and `drawImage` operations.